app/screens/Recognition/RecognitionAcknowledgement/RecognitionAcknowledgement.tsx

```diff
@@ -0,0 +1,65 @@
+import _ from 'lodash';
+import React, { useCallback } from 'react';
+import { Image, SafeAreaView, View } from 'react-native';
+
+import { IRecognitionAcknowledgementScreenProps } from './RecognitionAcknowledgement.types';
+import { styles } from './RecognitionAcknowledgement.styles';
+import {
+  BottomButtonView,
+  ExtendedText,
+  MainHeader,
+} from '../../../components';
+import { generalStyles } from '../../../utils/styles';
+import { IMAGES } from '../../../assets';
+import { useTranslation } from 'react-i18next';
+import { RECOGNITION_ACKNOWLEDGEMENT_DATA } from './RecognitionAcknowledgement.data';
+import { useAppSelector } from '../../../hooks';
+
+export const RecognitionAcknowledgementScreen: React.FC<IRecognitionAcknowledgementScreenProps> =
+  ({ navigation, route }) => {
+    const { type } = route.params.data;
+
+    const { t } = useTranslation();
+
+    const parentNickname = useAppSelector(
+      state => state.user.child?.nickname,
+    ) as string;
+    const childNickname = useAppSelector(
+      state => state.user.parent?.nickname,
+    ) as string;
+
+    const correctTitle = useCallback(() => {
+      const findedTitle =
+        _.find(RECOGNITION_ACKNOWLEDGEMENT_DATA, item => item.type === type)
+          ?.title ?? '';
+      return (
+        parentNickname + t('labels.and') + childNickname + ' ' + t(findedTitle)
+      );
+    }, [childNickname, parentNickname, t, type]);
```

Comment:
rename this function properly
if its function it should has name like `get...` `set...` `save...` (action prefix)

Reply:
also give it proper naming, not just `getCorrectTitle`
`correct` for what? do we have `incorrect` title? wtf?